### PR TITLE
Update Korean translation

### PR DESCRIPTION
**Pull request by Johnmacrocraft**
### Description
Updates Korean translation. Below is list what got changed.
Installer kor.ini: Genisys to PocketMine-MP in usage_info part, it's not
supposed to be changed.
Main kor.ini: Update Korean translation, now 100% completely translated
unless there's new string.
genisys_kor.yml: Update Korean translation

### Reason to modify
It has update of Korean translations (Installer kor.ini, Main kor.ini and genisys_kor.yml)

### Tests & Reviews
Tested and it works fine.

### CODE REVIEW

**Comment by Johnmacrocraft** [comment]
There's some stuffs which needs to be added from origin Genisys - DO NOT pull request it for now.

File: src/pocketmine/resources/genisys_kor.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 #Version 파일의 버전
 config:
- version: 20
+ version: 19
```
Comment (dktapps) [inline src/pocketmine/resources/genisys_kor.yml]: This is incorrect.

File: src/pocketmine/resources/genisys_kor.yml [hunk 2]
```diff
@@ -133,7 +133,35 @@ dserver:
...
+ creeper: true
+ #철 골램의 AI
+ iron-golem: true
+ #눈 골렘의 AI
+ snow-golem: true
+ #크리퍼가 폭발했을 때 블록 파괴
+ creeper-explode-destroy-block: false
+ #몹 자동 생성을 활성화 하는 경우 고르세요
+ mobgenerate: false
+
```
Comment (dktapps) [inline src/pocketmine/resources/genisys_kor.yml]: These properties do not exist anymore.

**Comment by Johnmacrocraft** [comment]
@dktapps I know I'm not allowed to ask something on pull request (I read pull request rule), but what does strength in cave command do? Does it means strength like strength effect? Sorry for asking, but I can't really figure out what it does, I even referenced other translate!

**Comment by dktapps** [comment]
> I know I'm not allowed to ask something on pull request
then why are you asking? (^_^)

**Comment by Johnmacrocraft** [comment]
@dktapps Huh, ok XD

**Comment by Johnmacrocraft** [comment]
@dktapps Anyway I fixed problems which you mentioned on top above, translated strength word. So could you consider merging this? Thanks.

**Comment by Johnmacrocraft** [comment]
@dktapps Not now, sorry for delay.

**Comment by Johnmacrocraft** [comment]
@dktapps Anyway I fixed problems which you mentioned on top above, and translated **strength** word in main kor.ini. Could you consider merging this? Thanks.

**Comment by dktapps** [comment]
Maybe if you stop spamming comments and squash your commits. You're getting irritating.

**Comment by Johnmacrocraft** [comment]
@dktapps Ok, I'll try to comment less and do pull request without problems. Sorry for that.

**Comment by Johnmacrocraft** [comment]
Because something went wrong, I'm sorry to say this but I need close it and create new pull request. It's kind of spammy stuff but there's nothing I can do (Maybe there's way but for some reason I deleted my Genisys fork and reforked.) Sorry again.

**Pull request closed without merging**